subprojects/android-test/ui-testing-maps/src/main/kotlin/com.avito.android.test.maps/GoogleMapFragmentElement.kt

```diff
@@ -1,24 +0,0 @@
-package com.avito.android.test.maps
-
-import androidx.annotation.IdRes
-import com.avito.android.test.maps.provider.FragmentGoogleMapProvider
-import com.avito.android.test.maps.provider.GoogleMapProvider
-import com.avito.android.test.page_object.PageObject
-import java.util.concurrent.TimeUnit
-
-class GoogleMapFragmentElement(
```

Comment:
Removed the whole module because of multiple reasons:

- It hasn't been used for a long time in tests
- It has been written non-conventionally - imperative logic instead of ViewAction/ViewAssertions. It could lead to bugs. See PR 20243 in the internal avito repository with the same issue for Yandex maps.

Thus, see no reason to fix it and support it. It's better to rewrite it from scratch if needed.

Reply:
Yep, totally agree